### ISSUE TITLE
[FEATURE REQUEST] Add name generator to the session, namely, the `Alias` command

### ISSUE THREAD

**Issue by felixguerrero12** [WangYihang/Platypus#12]
#### Description
```
Use a name generator to create a verb + an object to create an easily identifiable session. 
This will allow brains to separate session based on the name. 
Adding a hash next to the ops name will secure the possibility of same code names.
```

#### Reproduce
1. `go run platypus.go`
2. `Run 0.0.0.0 8080`
...

#### Expected behavior
Golden Duck - [f4f2b36f04c6e688fb210df160e5acd8]

#### Current behavior
f4f2b36f04c6e688fb210df160e5acd8

#### Environments
 - Standard

**Comment by WangYihang** [comment]
Solved. Now you can use `Alias` to rename a session and use `Jump [Hash | Name]` to switch among different sessions fluently.
This feature will be released in v1.2.9.
![image](https://user-images.githubusercontent.com/16917636/111896009-93b48f00-8a51-11eb-8179-3bcbfca7ca3c.png)

**Comment by WangYihang** [comment]
[v1.2.9](https://github.com/WangYihang/Platypus/releases/tag/v1.2.9) released. 
Feel free to have a try. Thanks a lot for your advice, bro!
